qemu/tests/balloon_check.py

```diff
@@ -148,6 +148,19 @@ def run_balloon_sub_test(self, test, params, env, test_tag):
             qemu_quit_after_test = 0
         return qemu_quit_after_test
 
+    def wait_for_balloon_complete(self):
+        """
+        Wait until guest memory don't change after do balloon
+        """
+        logging.info("Wait until guest memory don't change")
+        gmem = self.get_memory_status()
+        while True:
+            time.sleep(30)
+            cur_gmem = self.get_memory_status()
+            if abs(cur_gmem - gmem) < 100:
+                break
+            gmem = cur_gmem
```

Comment:
Thanks for your code. I've updated with the approach and shorter interval. As for the delta, when get windows guest memory by systeminfo, it increases the memory in use by dozens of MB.  I think 100M is OK here.